MapleServer2/Enums/SkillEnums.cs

```diff
@@ -11,7 +11,8 @@ public enum DamageType : byte
 {
     None = 0x00,
     Physical = 0x01,
-    Magic = 0x02
+    Magic = 0x02,
+    Primary = 0x03
```

Comment:
perhaps drop a comment on what primary means? i would think a primary skill can also be magic/physical.